independent-publisher-connectors/Tax ID Pro/readme.md

```diff
@@ -0,0 +1,24 @@
+# Tax ID Pro
+Tax ID Pro makes it easy to validate over 200 different tax ID formats for over 100+ countries.
+
+## Publisher: Fördős András
+
+## Prerequisites
+There are no prerequisites needed for this connector.
+
+## Obtaining Credentials
+You need to have an API key, for which you must sign for the service. There is a free plan available, follow the instructions here: [https://taxid.pro/docs/getting-started](https://taxid.pro/docs/getting-started)
+
+## Supported Operations
+### Validate 
+The Tax ID Pro API makes it easy to validate tax ID numbers for over 100 countries.
+### Lookup
+The lookup endpoint first validates the tax number, then enriches the response with additional data.
+
+## Known Issues and Limitations
+There are no known issues at this time.
+
+The connector does not implement the batch endpoint from the udnerlying service, while there might be missing parameters and properties too. If you miss anything, get in touch and let's figure out together how to bring in the missing pie
```

Comment:
Can you please correct the typo "udnerlying"?

Reply:
updated